src/components/ebay-textbox/examples/09-ds6-suported-icon/template.marko

```diff
@@ -0,0 +1 @@
+<ebay-textbox icon-name="messages" placeholder="email" />
```

Comment:
Sorry for the mispelling :P And I'm not sure why I didn't get a lint error. This will be fixed in my next commit for this and the other files as well.